changes/v5.0.10-en.md

```diff
@@ -9,6 +9,8 @@
 
 - Update `gen_rpc` library to version 3.0 [#9187](https://github.com/emqx/emqx/pull/9187).
 
+- Preserve MQTTv5 properties in Rules/Bridges [#9240](https://github.com/emqx/emqx/pull/9240).
```

Comment:
sorry, this has missed the v5.0.10 train.
please add it to v5.0.11-en.md (rebase master branch).